data/data/openstack/bootstrap/main.tf

```diff
@@ -1,3 +1,36 @@
+resource "openstack_networking_port_v2" "bootstrap_port" {
+  name = "${var.cluster_id}-bootstrap-port"
+
+  admin_state_up     = "true"
+  network_id         = var.private_network_id
+  security_group_ids = [var.master_sg_id]
+  tags               = ["openshiftClusterID=${var.cluster_id}"]
+  extra_dhcp_option {
```

Comment:
I think a space would be better right above

Reply:
done